src/backend/libs/efiling-facade-client/src/main/java/ca/bc/gov/open/jag/efilingfacadeclient/service/EfilingFacadeService.java

```diff
@@ -0,0 +1,44 @@
+package ca.bc.gov.open.jag.efilingfacadeclient.service;
+
+import ca.bc.gov.open.jag.ag.csows.filing.FilingPackage;
+import ca.bc.gov.open.jag.ag.csows.filing.SubmitFiling;
+import ca.bc.gov.open.jag.ag.csows.filing.SubmitFilingResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import org.springframework.ws.client.core.support.WebServiceGatewaySupport;
+
+import java.math.BigDecimal;
+import java.util.UUID;
+
+@Service
+public class EfilingFacadeService extends WebServiceGatewaySupport {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(EfilingFacadeService.class);
+
+    public BigDecimal SubmitFiling(String strFileInfo) {
```

Comment:
should that not be a `void` and just let it throw `runtimeException` ?

Reply:
The SubmitFiling SOAP call actually returns a BigDecimal.